docs/001_develop/03_client-capabilities/008_interaction/date-picker.mdx

```diff
@@ -0,0 +1,283 @@
+            }
+        }
+        ```
+    </TabItem>
+    <TabItem value="react">
+        Declaration
+        ```html
+        <rapid-date-picker></rapid-date-picker>
+        ```
+
+        Usage
+        ```tsx
+        export function MyComponent() {
+
+          const datePickerRef = useRef(null);
+          const [value, setValue] = useState('Not changed');
+          const handleValueChanged = (value) => {
+            setValue(value.detail);
+          }
+
+          useEffect(() => {
+            if (datePickerRef.current) {
+              datePickerRef.current.addEventListener('value-changed', handleValueChanged);
+            }
+          }, []);
+
+          return (
+            <rapid-date-picker ref={datePickerRef}>
+            </rapid-date-picker>
+          );
+        }
+        ```
+    </TabItem>
+    <TabItem value="angular">
+      Usage
+      ```typescript
+      import { AfterViewInit, Component, CUSTOM_ELEMENTS_SCHEMA, ElementRef, ViewChild } from '@angular/core';
+
+      @Component({
+      selector: 'date-picker-example',
+      template: `
+        <rapid-date-picker (value-changed)="handleValueChanged($event)">
+        </rapid-date-picker>
+        `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      })
+      export class DatePickerExampleComponent implements AfterViewInit {
+          valueChanged(e) {
+            console.log(e.detail);
+          }
+      }
+      ```
+    </TabItem>
+</Tabs>
+
+## DOM API
+
+The sections below provide property and attribute binding examples for Genesis Component syntax. The HTML closing tag is not included.
+
+### Attributes
+
+<table>
+    <thead>
+        <tr>
+            <th>Attribute</th>
+            <th>Type</th>
+            <th>Use</th>
+            <th>Example</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>format</td>
+            <td><code>string</code></td>
+            <td>The date format shown defaults to <code>MM-DD-YYYY</code> and can accept any <a href="https://day.js.org/docs/en/display/format">valid dayjs format</a></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker format="DD-MM-YYYY">
+            ```
+
+            </td>
+        </tr>
+        <tr>
+            <td>lang</td>
+            <td><code>string</code></td>
+            <td>Determines the defuault locale. Defaults to <code>en-US</code> and can accept any <a href="https://day.js.org/docs/en/i18n/changing-locale">valid day js lang</a></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker lang="en-US">
+            ```
+
+            </td>
+        </tr>
+        <tr>
+            <td>visible</td>
+            <td><code>boolean</code></td>
+            <td>Displays the calendar by default. Defaults to <code>false</code></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker visible>
```

Comment:
Might be nice to show an example for the boolean ones using the dynamic `?` template syntax